### ISSUE TITLE
Relabel term for BCL10 gene

### ISSUE THREAD

**Issue by mnzelnic** [monarch-initiative/mondo#6186]
**Mondo term (ID and Label):**
Immunodeficiency 37 MONDO:0014491

**Suggested new label:**
Combined Immunodeficiency due to BCL10 Deficiency

ORCID ID:
0000-0002-4455-3112

**_Optional_: Any additional information (like supporting evidence, PubMed ID, etc.)**

BCL10 was first reported in relation to autosomal recessive combined immunodeficiency in 2014 (Torres et al., PMID: 25365219). Three variants (nonsense and splice donor) that have been reported in 3 probands in 3 publications are included in this curation (PMIDs: 25365219, 32008135, 34868072). The mechanism of pathogenicity appears to be loss of function. Individuals affected with combined immunodeficiency due to BCL10 deficiency present in infancy with severe, recurrent infections. While more evidence is needed to establish the full phenotype, patients reported to date have normal total T cell counts but a deficit of memory T cells combined with impaired effector T cell functions, hypogammaglobulinemia, and decreased memory B cells. Low counts of other T cell subsets and NK cells have been described. Heterozygous carriers are reported as unaffected (PMID: 25365219, 34868072).
This gene-disease association is supported by biochemical studies, protein interactions, functional and rescue assays, as well as animal models (PMIDs: 10400625, 24074955, 25365219, 32008135, 11163238, 12910267). BCL10 is a part of the CARD11/9–BCL10–MALT1 (CBM) complex. It acts as a signaling molecule transducing the triggering of cell surface antigen receptors to NF-kappaB activation (PMID: 10400625, 24074955). We have previously established other genes within this complex including CARD11 as definitively associated with severe combined immune deficiency. Affected patient cells show altered NF-kappaB signaling that is rescued by wild type (PMID: 25365219, 32008135). BCL10 knockout mice have B cell lymphopenia, reduced B cell proliferation, hypogammaglobulinemia, and impaired immune responses consistent with combined immunodeficiency (PMID: 11163238, 12910267).

In summary, BCL10 is definitively associated with combined immunodeficiency due to BCL10 deficiency. This has been repeatedly demonstrated in both the research and clinical diagnostic settings and has been upheld over time. This classification was approved by the ClinGen SCID/CID GCEP on the meeting date of December 15, 2022 (SOP Version 9).

@marwa-elnagheeb

**Comment by nicolevasilevsky** [comment]
In accordance to the [Mondo disease naming conventions](https://mondo.monarchinitiative.org/pages/disease-naming/), we aim to keep the terms from the OMIM series consistent in the branch We can add a synonym and a "ClinGen preferred" tag to this term and I will add your notes above as a comment on the term.

**Comment by mnzelnic** [comment]
@courtneythaxton

**Comment by courtneythaxton** [comment]
@nicolevasilevsky 

As a follow up to our discussion, would it be possible for you or a Mondo representative to attend some of the SCID GCEP calls to discuss the groups efforts and timeline to update the immunodificiency nomenclature? I believe Michelle (Shelly) has sent a follow email to you to that effect. This group would like to work on the entire immunodificiency phenotypic series and has a desire to use explicit dyadic terms, where appropriate.

Thanks!
@courtneythaxton

**Comment by nicolevasilevsky** [comment]
@courtneythaxton I don't see an email from Shelly. When is this meeting?

**Comment by courtneythaxton** [comment]
@mnzelnic Would you be able to resend the email to Mondo with your request to meet, or add the meeting times below, thanks!

Courtney

**Comment by sabrinatoro** [comment]
@mnzelnic and @nicolevasilevsky : I also have conflicts. Could we please look for a different day/time? Thank you.

**Comment by mnzelnic** [comment]
Hi there,

What are your emails? I do not have them.

Best,

Shelly


Michelle Zelnick, MS, CGC

ClinGen SCID Coordinator

Department of Allergy and Immunology/Neurology

Baylor College of Medicine

(she/her/hers)

[cid:63708154-16a6-4497-aa18-97ad1027cc19]​ [cid:2179a750-6cfc-4b5d-9f6b-19f53a3ec8fa]
________________________________

**Comment by nicolevasilevsky** [comment]
nicole {at} tislab {dot} org
sabrina {at} tislab {dot} org

thanks!
